dags/fission_experiment_monitoring.py

```diff
@@ -25,25 +25,17 @@
         schedule_interval="0 2 * * *",
         default_args=default_args) as dag:
 
-    wait_for_copy_deduplicate_main_ping = ExternalTaskSensor(
-        task_id="wait_for_copy_deduplicate_main_ping",
-        external_dag_id="copy_deduplicate",
-        external_task_id="copy_deduplicate_main_ping",
-        execution_delta=datetime.timedelta(hours=1),
+    wait_for_main_nightly = ExternalTaskSensor(
+        task_id="wait_for_main_nightly",
+        external_dag_id="bqetl_main_summary",
+        external_task_id="telemetry_derived__main_nightly__v1",
+        execution_delta=datetime.timedelta(hours=2),
```

Comment:
```suggestion
```
`bqetl_main_summary` is also scheduled for `0 2 * * *` so `execution_delta` shouldn't be necessary here. It might however be good to schedule this DAG a little later, maybe `0 3 * * *`, to ensure `bqetl_main_summary` already finished. Otherwise, since both DAGs start at the same time, this one will have to wait.

Reply:
Good idea, thanks!